islands/src20/cards/SRC20BaseCard.tsx

```diff
@@ -2,6 +2,22 @@ import { useState } from "preact/hooks";
 import { SRC20Row } from "$globals";
 import { convertToEmoji } from "$lib/utils/emojiUtils.ts";
 
+function splitTextAndEmojis(text: string): { text: string; emoji: string } {
```

Comment:
perhaps better placed in /lib/utils/emojiUtils.ts or utilize some of the existing functions there ?